src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -562,19 +562,72 @@ static unique_ptr<Picture> _imageBuildHelper(SvgNode* node, const Box& vBox, con
 
 static unique_ptr<Scene> _useBuildHelper(const SvgNode* node, const Box& vBox, const string& svgPath, bool* isMaskWhite)
 {
+    unique_ptr<Scene> finalScene;
```

Comment:
If overflow(and x, y) attribute is declared in symbol, `finalScene` does not correspond to the if condition below.
I don't know if the overflow attribute is fully supported in the chrome browser, but it seems to be different from what I expected.

``` html
<svg viewBox="0 0 80 20" xmlns="http://www.w3.org/2000/svg">
  <!-- Our symbol in its own coordinate system -->
  <symbol id="myDot" width="10" height="10" viewBox="0 0 2 2" overflow="visible">
    <circle cx="1" cy="1" r="1" />
  </symbol>

   <!-- A grid to materialize our symbol positioning -->
  <path d="M0,10 h80 M10,0 v20 M25,0 v20 M40,0 v20 M55,0 v20 M70,0 v20" fill="none" stroke="pink" />

  <!-- All instances of our symbol -->
  <use href="#myDot" x="5"  y="5" style="opacity:1.0" />
  <use href="#myDot" x="20" y="5" style="opacity:0.8" />
  <use href="#myDot" x="35" y="5" style="opacity:0.6" />
  <use href="#myDot" x="50" y="5" style="opacity:0.4" />
  <use href="#myDot" x="65" y="5" style="opacity:0.2" />
</svg>

```

Reply:
and what would you expect to happen in this specific example? in this case the overflow attrib, set or not, doesn't change the result. try to set `r="2"`:
`    <circle cx="1" cy="1" r="2" />`

Reply:
> If overflow(and x, y) attribute is declared in symbol, `finalScene` does not correspond to the if condition below. 

Could you please rephrase this. I don't know what do you mean.
FinalScene is a `tvg::Scene' and `overflow` is a property of an `SvgSymbolNode`. in an `SvgUseNode` I added a pointer to the corresponding `SvgNode` (set only if a `symbol` was used to define a `use` element ).

Reply:
The result in thorvg and in chrome are different.

```html
<svg viewBox="0 0 80 20" xmlns="http://www.w3.org/2000/svg">
  <!-- Our symbol in its own coordinate system -->
  <symbol id="myDot" width="10" height="10" viewBox="0 0 2 2" overflow="visible">
    <circle cx="1" cy="1" r="2" />
  </symbol>

   <!-- A grid to materialize our symbol positioning -->
  <path d="M0,10 h80 M10,0 v20 M25,0 v20 M40,0 v20 M55,0 v20 M70,0 v20" fill="none" stroke="pink" />

  <!-- All instances of our symbol -->
  <use href="#myDot" x="5"  y="5" style="opacity:1.0" />
  <use href="#myDot" x="20" y="5" style="opacity:0.8" />
  <use href="#myDot" x="35" y="5" style="opacity:0.6" />
  <use href="#myDot" x="50" y="5" style="opacity:0.4" />
  <use href="#myDot" x="65" y="5" style="opacity:0.2" />
</svg>

```
[declare overflow attribute]
thorvg
![image](https://user-images.githubusercontent.com/7413838/153973693-0d1ba323-ef0e-4fd4-9e1f-28ca60251799.png)

chrome
![image](https://user-images.githubusercontent.com/7413838/153973732-8a04db0a-4c82-4c55-8ca6-e4f2cf18b56a.png)

Reply:
The overflow attribute is related to clipping of the viewbox area.
So I'm not sure, but I think the code should be changed like below.

```cpp
    if (node->node.use.symbol){// && !node->node.use.symbol->node.symbol.overflowVisible) {
        auto symbol = node->node.use.symbol->node.symbol;

        Matrix mViewBox = {1, 0, 0, 0, 1, 0, 0, 0, 1};
        if ((!mathEqual(symbol.w, symbol.vw) || !mathEqual(symbol.h, symbol.vh)) && symbol.vw > 0 && symbol.vh > 0) {
            auto sx = symbol.w / symbol.vw;
            auto sy = symbol.h / symbol.vh;
            if (symbol.preserveAspect) {
                if (sx < sy) sy = sx;
                else sx = sy;
            }

            auto tvx = symbol.vx * sx;
            auto tvy = symbol.vy * sy;
            auto tvw = symbol.vw * sx;
            auto tvh = symbol.vh * sy;
            if (tvw > tvh) tvy -= (symbol.h - tvh) * 0.5f;
            else tvx -= (symbol.w - tvw) * 0.5f;

            mViewBox = {sx, 0, -tvx, 0, sy, -tvy, 0, 0, 1};
        } else if (!mathZero(symbol.vx) || !mathZero(symbol.vy)) {
            mViewBox = {1, 0, -symbol.vx, 0, 1, -symbol.vy, 0, 0, 1};
        }

        // mSceneTransform = mUseTransform * mViewBox
        Matrix mSceneTransform = mathMultiply(&mUseTransform, &mViewBox);
        scene->transform(mSceneTransform);
        auto root = Scene::gen();

        if (!node->node.use.symbol->node.symbol.overflowVisible) {
            auto viewBoxClip = Shape::gen();
            viewBoxClip->appendRect(0, 0, symbol.w, symbol.h, 0, 0);
            // mClipTransform = mUseTransform * mSymbolTransform
            Matrix mClipTransform = mUseTransform;
            if (node->node.use.symbol->transform) {
                mClipTransform = mathMultiply(&mUseTransform, node->node.use.symbol->transform);
            }
            viewBoxClip->transform(mClipTransform);

            auto compositeLayer = Scene::gen();
            compositeLayer->composite(move(viewBoxClip), CompositeMethod::ClipPath);
            compositeLayer->push(move(scene));

            root->push(move(compositeLayer));
        } else {
            root->push(move(scene));
        }

        finalScene = move(root);
    } else if (node->node.use.x != 0.0f || node->node.use.y != 0.0f) {
        scene->transform(mUseTransform);
        finalScene = move(scene);
    }
```


result
![image](https://user-images.githubusercontent.com/7413838/153974757-193f685e-793e-4746-8ce1-8794e70bf00b.png)

Reply:
thx for that! I pushed a commit

Reply:
I checked ! thx :)